### DISCUSSION TITLE
Recommendations on disabled items within a dropdown

### DISCUSSION THREAD

**Discussion by mindytom** [twilio-labs/paste#1071] [Q&A] [answered]
Hey DSYS! 

My team (including @cfree) is working on a dropdown and one of the item is to "Duplicate". This item can be disabled if the user have already maxed out the number of variations for that email. We're thinking of showing some text like "You have the maximum of 6 versions. Please delete one to duplicate this version" so users understand why "Duplicate" is disabled. 

Was wondering if DSYS have any recommendations on how to show this additional information? We're considering having an icon with the tooltip next to Duplicate dropdown item but didn't see examples of that on the Paste website. Also, found that [Neilson Norman Group](https://www.nngroup.com/articles/drop-down-menus/) recommends having short balloon help message for disabled items in dropdown. 

<img width="532" alt="Screen Shot 2021-01-06 at 9 04 00 AM" src="https://user-images.githubusercontent.com/65037539/103798107-21a9dd80-4ffe-11eb-9c73-ce5c00bfd14b.png">

**Comment by richbachman** [comment]
Hi @mindytom 

Would you mind sharing a mockup so the team can better answer this one? Thanks!

**Reply by mindytom** [reply]
<img width="1918" alt="Dropdown" src="https://user-images.githubusercontent.com/65037539/103818992-6cd3e880-501e-11eb-85ff-880f68ac7199.png">

<img width="1919" alt="Dropdown-Tooltip" src="https://user-images.githubusercontent.com/65037539/103818982-66457100-501e-11eb-92dc-6c66f46129d9.png">

**Comment by richbachman** [comment, marked as answer]
Thanks for sending the screenshots @mindytom.

I'm mainly speaking to the tooltip within a dropdown menu here. I don't think thats best for accessibility reasons. I'd suggest using help text here where you can explain the limits. Here's an example of help text with a combobox: https://paste.twilio.design/components/combobox#autocomplete-combobox

If that's not quite enough, you could use a tooltip or popover on the field itself in either prefix or suffix. Something similar to this: https://paste.twilio.design/components/combobox#combobox-with-add-ons-prefix-suffix-text-or-icons (Not exact, since theres not tooltip).

**Comment by SiTaggart** [comment]
A short note on why the advice from Neilson Norman Group might not be something to follow.

Disabled controls are entirely inactive. You can't click on them, focus on them, or technically hover on them. They are effectively removed from the page. 

This means adding something interactive like a tooltip to a disabled control, is not possible because you can't interact with the item to make it show.

**Comment by mindytom** [comment]
Thanks for the quick reply and clarifying the best practice here! We won't be using a tooltip for this.
